data/json/mutation_category.json

```diff
@@ -144,7 +144,7 @@
         "name" : "Medical","//" : "2-6 pain, same as Alpha--since specifically intended for medical applications.",
         "category" : "MEDICAL",
         "mutagen_message" : "You can feel the blood rushing through your veins and a strange, medicated feeling washes over your senses.",
-        "iv_message" : "You can feel the blood in your medication stream.  It's a strange feeling.",
+        "iv_message" : "You can feel the medication in your blood stream.  It's a strange feeling.",
```

Comment:
Oh. I get how this could be a joke but in the context it came up it really looks like a mistake. Especially since you get this message just from injecting, in my case it was the first time using a mutation on my current character. I guess it just needs to be rewritten somehow. Suggestions anyone?